packages/application-components/src/components/detail-pages/form-detail-page/form-detail-page.tsx

```diff
@@ -84,6 +84,10 @@ type FormDetailPageProps = {
    * Hides the form controls.
    */
   hideControls: boolean;
+  /**
+   * The Icon for the secondary button label
+   */
+  labelSecondaryButtonIcon?: ReactElement;
```

Comment:
@Rhotimee Remember to also document this in the component table props in the docs. Same for other affected components. Thanks

Reply:
I already did that in the PR.